spec/integration_spec.rb

```diff
@@ -1519,6 +1519,25 @@ class ForwardToReplicasTwo < ActiveRecord::Base
   end
 end
 
+describe 'Pagy' do
+  before(:all) do
+    require 'pagy'
+    AlgoliaSearch.configuration = { :application_id => ENV['ALGOLIA_APPLICATION_ID'], :api_key => ENV['ALGOLIA_API_KEY'], :pagination_backend => :pagy }
```

Comment:
Some tests after this one are failing, because the pagination backend is configured globally by this line, and the return value differs from other pagination backends (others return one value, while pagy returns 2).

I see that Pagy defaults to returning 2 values (Pagy object and the paginated list of items), so we should reset the pagination backend after these tests have run to make the other tests still pass.